assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -163,6 +155,25 @@ che.workspace.startup_debug_log_limit_bytes=10485760
 # This configuration is mainly required for workspace idling when the OpenShift OAuth is enabled.
 che.workspace.stop.role.enabled=true
 
+
+# Name of cofig map in Che server namespace with additional CA TLS certificates to be propogated into all user's workspaces.
```

Comment:
```suggestion
# Name of config map in Che server namespace with additional CA TLS certificates to be propagated into all user's workspaces.
```

propogated appears in other places as well.

Reply:
Thank, will fix.